rfcs/input-component.md

```diff
@@ -125,11 +129,11 @@ export default () => {
   Defines styles for inputs that are grouped with other controls.
 * `size: 'default' | 'compact'`
   Defines the size of an input control.
-* `onChange: Function` - Optional
+* `onChange: SyntheticInputEvent` - Optional
```

Comment:
this should still be a `Function` but maybe `(e: SyntheticInputEvent<HTMLInputElement>) => void`